components/Card/index.js

```diff
@@ -66,9 +91,26 @@ export default function Card({ initialItems, items }) {
           })}
         </ul>
       )}
+
       <ul style={{ listStyle: "none" }}>
```

Comment:
You are using styled-components. To keep it consistent please don't use inline styling and create a styled component instead.

Reply:
i changed this